DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h

```diff
@@ -221,6 +223,8 @@ struct DataRequest {
   void requestPrimaryVerterticesTMP(bool mc);
   void requestSecondaryVertertices(bool mc);
 
+  void requestCaloCells(bool mc);
```

Comment:
EMCAL cells and triggers were already implemented, see https://github.com/AliceO2Group/AliceO2/blob/7274c228d3335892a51987bb4dc420fa2e9bdfaa/DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h#L217
https://github.com/AliceO2Group/AliceO2/blob/7274c228d3335892a51987bb4dc420fa2e9bdfaa/DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h#L351
https://github.com/AliceO2Group/AliceO2/blob/7274c228d3335892a51987bb4dc420fa2e9bdfaa/DataFormats/Detectors/GlobalTracking/include/DataFormatsGlobalTracking/RecoContainer.h#L603-L604

Reply:
I am aware. This change should not have been pushed .. working on it. Thanks for spotting

Reply:
I removed the changes. Sorry again for the mess ... Note to self: never change branches when you still have an open pull request and then force push from the wrong branch. Now everything should be fine. Please let me know if you still spot something